omc3/segment_by_segment/propagables/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""
+Propagables
+-----------
+
+In the ``propagables`` module, the parameters that can be propagated
+through the segment, are defined. 
+Each propagable has a corresponding class, which contains the functions that
+describe the forward and backward propagation for the respective parameter.
+
+This module exposes the main classes for propagables, to avoid making the 
+imports so long and complicated.
+"""
+from omc3.segment_by_segment.propagables.abstract import Propagable  # probably only used for type-hinting
+from omc3.segment_by_segment.propagables.utils import PropagableBoundaryConditions   #noqa  -> expose
+from omc3.segment_by_segment.propagables.utils import PropagableColumns   #noqa  -> expose
+from omc3.segment_by_segment.propagables.phase import Phase
+from omc3.segment_by_segment.propagables.beta import BetaPhase
+from omc3.segment_by_segment.propagables.alpha import AlphaPhase
```

Comment:
We should make an `__all__` with things we want to publicly expose.

Reply:
Is this not only encouraging people to do `from propagables import *` ?
Also `__all__` here would just repeat all the things inside (do not want to hide anything)

Reply:
No, it does not encourage, it is a safeguard in case someone does. I believe IDEs also try to inspect __all__ if possible when suggesting completions. It's better practice